Modules/Core/ImageFunction/include/itkWindowedSincInterpolateImageFunction.h

```diff
@@ -343,14 +343,14 @@ class ITK_TEMPLATE_EXPORT WindowedSincInterpolateImageFunction : public Interpol
   TWindowFunction m_WindowFunction;
 
   /** Size of the offset table */
-  static constexpr unsigned int m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
+  static constexpr SizeValueType m_OffsetTableSize{ Math::UnsignedPower(m_WindowSize, ImageDimension) };
 
   /** The offset array, used to keep a list of relevant
    * offsets in the neihborhoodIterator */
-  unsigned int m_OffsetTable[m_OffsetTableSize];
+  SizeValueType m_OffsetTable[m_OffsetTableSize];
```

Comment:
Is this particular change really needed for VS2017? The error message is only about line 346, if I understand correctly.